waspc/cli/src/Wasp/Cli/Command/CreateNewProject/StarterTemplates.hs

```diff
@@ -7,78 +7,190 @@ module Wasp.Cli.Command.CreateNewProject.StarterTemplates
-        )
-
-localTemplates :: [StarterTemplate]
-localTemplates = [defaultStarterTemplate]
+type StartingInstructionsBuilder = String -> String
+
+{- HLINT ignore getTemplateStartingInstructions "Redundant $" -}
+
+-- | Returns instructions for running the newly created (from the template) Wasp project.
+-- Instructions assume that user is positioned right next to the just created project directory,
+-- whose name is provided via projectDirName.
+getTemplateStartingInstructions :: String -> StarterTemplate -> String
+getTemplateStartingInstructions projectDirName = \case
+  GhRepoStarterTemplate _ metadata -> _buildStartingInstructions metadata projectDirName
+  LocalStarterTemplate metadata -> _buildStartingInstructions metadata projectDirName
+  AiGeneratedStarterTemplate ->
+    unlines
+      [ styleText $ "To run your new app, do:",
+        styleCode $ "    cd " <> projectDirName,
+        styleCode $ "    wasp db migrate-dev",
+        styleCode $ "    wasp start"
+      ]
+
+getStarterTemplates :: [StarterTemplate]
+getStarterTemplates =
+  [ defaultStarterTemplate,
+    todoTsStarterTemplate,
+    openSaasStarterTemplate,
+    embeddingsStarterTemplate,
+    AiGeneratedStarterTemplate
+  ]
 
 defaultStarterTemplate :: StarterTemplate
-defaultStarterTemplate =
+defaultStarterTemplate = basicStarterTemplate
+
+{- HLINT ignore basicStarterTemplate "Redundant $" -}
+
+basicStarterTemplate :: StarterTemplate
+basicStarterTemplate =
   LocalStarterTemplate $
     DirBasedTemplateMetadata
-      { _name = "basic",
-        _path = "basic",
-        _description = "Simple starter template with a single page."
+      { _path = [reldir|basic|],
+        _name = "basic",
+        _description = "Simple starter template with a single page.",
+        _buildStartingInstructions = \projectDirName ->
+          unlines
+            [ styleText $ "To run your new app, do:",
+              styleCode $ "    cd " <> projectDirName,
+              styleCode $ "    wasp db start"
+            ]
       }
 
+{- HLINT ignore openSaasStarterTemplate "Redundant $" -}
+
+openSaasStarterTemplate :: StarterTemplate
+openSaasStarterTemplate =
+  simpleGhRepoTemplate
+    ("open-saas", [reldir|.|])
+    ( "saas",
+      "Everything a SaaS needs! Comes with Auth, ChatGPT API, Tailwind, Stripe payments and more."
+        <> " Check out https://opensaas.sh/ for more details."
+    )
+    ( \projectDirName ->
+        unlines
+          [ styleText $ "To run your new app, do:",
+            styleCode $ "    cd " <> projectDirName FP.</> "app",
+            styleCode $ "    wasp db start",
+            styleText $ "Then open new terminal window/tab in that same dir and do:",
+            styleCode $ "    wasp db migrate-dev",
+            styleCode $ "    cp .env.server.example .env.server",
+            styleCode $ "    wasp start",
+            styleText $ "",
+            styleText $ "Check the README for additional guidance and the link to docs!"
+          ]
```

Comment:
```suggestion
          [ styleText $ "To run your new app, do:",
            styleCode $ "    cd " <> projectDirName FP.</> "app",
            styleCode $ "    wasp db start",
            styleText $ "",
            styleText $ "Then open new terminal window/tab in that same dir and do:",
            styleCode $ "    wasp db migrate-dev",
            styleCode $ "    cp .env.server.example .env.server",
            styleCode $ "    wasp start",
            styleText $ "",
            styleText $ "Check the README for additional guidance and the link to docs!"
          ]
```

Reply:
I did this in a different way, commented above.